### DISCUSSION TITLE
Problem, when editing model's 'title' that is not translatable,

### DISCUSSION THREAD

**Discussion by ResetButton** [area17/twill#893] [General]
Hello
If model field 'title' set as not translatable, 
when creating item - this field works fine http://prntscr.com/scgrkx and item is saving as usual
when I click on this item's title to edit, after saving - this field it shows as translatable http://prntscr.com/scgtdw and fails saving.

Code

**Model**
`class Metadata extends Model 
{
    use HasTranslation, HasMedias, HasFiles, HasRevisions;

    protected $fillable = [
        'title',
        'published',
        'seo_title',
        'seo_description',
        'content'

    ];

    public $translatedAttributes = [
        'seo_title',
        'seo_description',
        'content'
    ];
}`

**Migration**
`Schema::create('metadata', function (Blueprint $table) {
            // this will create an id, a "published" column, and soft delete and timestamps columns
            createDefaultTableFields($table);
            $table->string('title', 200)->nullable();

            // add those 2 columns to enable publication timeframe fields (you can use publish_start_date only if you don't need to provide the ability to specify an end date)
            // $table->timestamp('publish_start_date')->nullable();
            // $table->timestamp('publish_end_date')->nullable();
        });

        Schema::create('metadata_translations', function (Blueprint $table) {
            createDefaultTranslationsTableFields($table, 'metadata');

            $table->string('seo_title', 255)->nullable();
            $table->string('seo_description', 255)->nullable();

            $table->text('content')->nullable();
        });`

Any suggestions?

**Comment by nacelley** [comment]
Replied in the topic you created on [Spectrum](https://spectrum.chat/twill/help/bug-title-that-set-in-not-translatable-makes-translatatable-when-editing~a2f3a2bd-71b1-4d85-ac3c-17cd9312a091)

**Comment by ResetButton** [comment]
Copy @nacelley answer here
----
Check if you didn't override the create blade for the module or didn't override the passed translateTitle.
A translated param for the input with set true makes the field translatable in the UI.
A default value of it is the value of the $translateTitle that is being decided on the result of the isTranslatable function.
For index it is being passed automatically.
For form if it is not passed by you, then if you use translations, it would be marked as translatable when you use translations in the module by default (I don't like the approach by the way).
`$translateTitle = $translateTitle ?? $translate ?? false;`
Made a PR to pass it automatically.

For now, you need to pass it through the formData function in your module's controller or just override the create blade.

> public function formData($request)
    {
        return [
            'translateTitle' => $this->titleIsTranslatable(), // you could return false instead of this additional call if you wish
        ];
    }
